srcdeps-core/src/main/java/org/srcdeps/core/impl/builder/ShellBuilder.java

```diff
@@ -134,4 +152,22 @@ protected String locateExecutable(BuildRequest request) {
         return result;
     }
 
+    /**
+     * Returns a new {@link Map} that contains environment variables combined from the following sources:
+     * <ul>
+     * <li>{@link #getDefaultBuildEnvironment()} (if {@code request.isAddDefaultBuildEnvironment()} return
```

Comment:
Well, this means that it just returns either an empty map, or the map from the default build environment :) There's no merging happening here. Or am I missing something?

Reply:
Looks like a bug. The code should be merging the env from the request with the default env. I filed an issue https://github.com/srcdeps/srcdeps-core/issues/14